app/Data/Geolocation.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Data;
+
+/**
+ * Utility class for a geolocation (point).
```

Comment:
Call it a ValueObject in the comments. Design wise, that is what it is called ;) 
Very nice implementation btw.